src/ocm/services/apis/ClustersAPI.ts

```diff
@@ -156,6 +156,12 @@ const ClustersAPI = {
     );
   },
 
+  addHosts(clusterId: Cluster['id']) {
```

Comment:
I think a better name would be `allowAddHosts` since it's not actually adding any host.